setup.py

```diff
@@ -5,14 +5,14 @@
 
 DEPENDENCIES = [
     "APScheduler",
-    "co2-tracker-utils",
     "dash",
     "dash_bootstrap_components",
     "dataclasses",
     "fire",
     "pandas",
     "pynvml",
     "requests",
+    "typing>=3.7.4; python_version<'3.5'",
```

Comment:
why python version<3.5 ?
In the docs, the recommended version is 3.6

Reply:
Ok good catch, let's remove it